guides/common/modules/con_managing-alternate-content-sources.adoc

```diff
@@ -2,9 +2,28 @@
 = Managing Alternate Content Sources
 
 Alternate content sources define alternate paths to download content during synchronization.
-The content itself is downloaded from the alternate content source and only the metadata is downloaded from the upstream URL.
+The content itself is downloaded from the alternate content source, while the metadata is downloaded from the {ProjectServer} or the upstream URL, depending on the configuration.
 You can use alternate content source to speed up synchronization if the content is located on the local filesystem or on a nearby network.
 You can set up alternate content sources for {ProjectServer} and {SmartProxy}.
 You must refresh the alternate content source after creation or after making any changes.
 A weekly cron job refreshes all alternate content sources.
 You can also refresh the alternate content sources manually using the {ProjectWebUI} or the Hammer CLI.
+
+There are three types of alternate content sources:
+
+Custom::
+Custom alternate content source downloads the content from any upstream repository or from the local filesystem.
+
+Simplified::
+Simplified alternate content source copies the upstream repo information from your {ProjectServer}.
+You can use simplified alternate content sources in situations where the connection from your {SmartProxy} to the upstream repo is faster than from your {ProjectServer}.
+ifdef::satellite[]
+Selecting the Red Hat products when creating a simplified alternate content source will download the content to the {SmartProxies} from the {Team} CDN.
+endif::[]
+
+RHUI::
+RHUI alternate content source downloads content from the Red Hat Update Infrastructure server in your environment.
+ifdef::satellite[]
+{Team} only supports RHUI type alternate content source with RHUI 4 or above.
+{Team} also does not support configuration of RHUI alternate content source with the {Team} provided RHUI instances for AWS.
```

Comment:
Rather than saying that AWS RHUI is not supported, it might be better to say that only the default RHUI installation is supported. There might in the future be other modes of "special" RHUI installations that Satellite also won't support. Here's a comment with the original explanation: https://bugzilla.redhat.com/show_bug.cgi?id=2001087#c6